From 99ca66b77aae6144e436ab2347b6411f43bdfb1b Mon Sep 17 00:00:00 2001
From: Venkateshwar Rao G <venkateshwar.rao.gannavarapu@xilinx.com>
Date: Wed, 21 Feb 2018 13:15:05 +0530
Subject: [PATCH 0282/1851] dt: bindings: display: xlnx: Adding gpio reset
 entry

commit 9a628a9116f3537d673d1ade6bf1452d422bd029 from
https://github.com/Xilinx/linux-xlnx.git

This patch adds the gpio reset entry for vpss scaler.

Signed-off-by: Venkateshwar Rao G <venkateshwar.rao.gannavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt      | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt
index 0e8b3854a48c..9d46dc28807b 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt
@@ -22,10 +22,13 @@ Required properties:
 - xlnx,pix-per-clk : The pixels per clock property of the IP.
   supported values are 1 and 2.
 
+- reset-gpios: Specifier for a GPIO that assert for VPSS Scaler reset.
+
 Example:
 	scaler@a0040000 {
 		compatible = "xlnx,vpss-scaler";
 		reg = <0x0 0xa0000000 0x0 0x40000>;
+		reset-gpios = <&axi_gpio_1 0x0 0x0>;
 		xlnx,num-hori-taps = <8>;
 		xlnx,num-vert-taps = <8>;
 		xlnx,pix-per-clk = <2>;
-- 
2.31.1

